tests/test_device.py

```diff
@@ -235,6 +236,17 @@ def prop(self, value):
     dev.sub_attrsig.put(0)
 
 
+def test_attribute_signal_attributeerror():
+    class MyDevice(Device):
+        sig1 = Component(AttributeSignal, 'unknownattribute')
+        sig2 = Component(AttributeSignal, '__class__.__name')
+
+    dev = MyDevice('', name='mydev')
+    pytest.raises(AttributeError, dev.sig1.get)
+    pytest.raises(AttributeError, getattr(dev.sig1, 'base'))
```

Comment:
I think this needs to be `pytest.raises(AtributeError, getattr, dev.sig1, 'base')`?